tests/integration/checkout/test_views.py

```diff
@@ -5,7 +5,7 @@
 
 
 class ThankYouViewTestCase(WebTestCase):
-    is_anonymous = False
+    is_anonymous = True
```

Comment:
Hm, aren't existing test methods supposed to be run for anonymous users?

Reply:
Yes, the test methods all use `OSCAR_ALLOW_ANON_CHECKOUT = True`, and make anonymous (without specifying a user) requests. So I am not sure why `is_anonymous` would be `False`?